src/scripts/main.js

```diff
@@ -1,3 +1,85 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const message = document.createElement('div');
+
+// message.style.top = `30px`;
+// message.style.right = `30px`;
+
+const urlList
```

Comment:
```suggestion
const DETAILS_URL
```
для таких констант краще використовуати верхній регістр